prbt_support/urdf/prbt.xacro

```diff
@@ -57,16 +57,13 @@ limitations under the License.
   <!-- If gripper given then... -->
   <xacro:if value="${gripper != ''}">
     <!-- name of the gripper -->
-    <xacro:arg name="gripper_name" default="$(arg robot_prefix)gripper"/>
+    <xacro:property name="gripper_name" value="$(arg robot_prefix)gripper"/>
 
     <xacro:include filename="$(find prbt_${gripper}_support)/urdf/${gripper}.urdf.xacro" />
```

Comment:
Imho we could use this (API-)change, to also make pg70.urdf.xacro a module (in a follow-up-PR). Why is this just a single include without instantiation?

Reply:
I am not sure. Do we follow any convention in the package `prbt_pg70_support`?